packages/expandable-search-bar/package.json

```diff
@@ -32,25 +32,21 @@
     "test:prune-snapshots": "karma start --prune-snapshots",
     "test:compatibility": "karma start --compatibility all --coverage",
     "test:compatibility:watch": "karma start --compatibility all --auto-watch=true --single-run=false",
-    "test:bs": "karma start karma.bs.config.js --compatibility all --coverage",
-    "site:build": "npm run storybook:build",
-    "storybook": "start-storybook -p 9001",
-    "storybook:build": "build-storybook -o _site -s storybook-static"
+    "test:bs": "karma start karma.bs.config.js --compatibility all --coverage"
+    
   },
   "dependencies": {
     "lit-element": "^2.2.1",
     "lit-html": "^1.0.0"
   },
   "devDependencies": {
-    "@open-wc/demoing-storybook": "^0.3.0",
```

Comment:
There should also be an update to the `yarn.lock` file when removing these. Did you run `yarn remove` or just delete these lines?

Reply:
yarn.lock updated